Vagrantfile

```diff
@@ -47,6 +50,26 @@ Vagrant.configure(2) do |config|
     do
         echo unset $LC  >> /home/vagrant/.bashrc
     done
+
+    # Install VirtualBox guest additions
+    cd /tmp
+    VBOX_VERSION=$1
+    VBOX_ADDITIONS_URL="http://download.virtualbox.org/virtualbox/VER/VBoxGuestAdditions_VER.iso"
+    sudo wget -q -c ${VBOX_ADDITIONS_URL//VER/$VBOX_VERSION} -O VBoxGuestAdditions_$VBOX_VERSION.iso
+    sudo mount VBoxGuestAdditions_$VBOX_VERSION.iso -o loop /mnt
+    expect -c '
+        set timeout -1;
+        spawn sudo sh /mnt/VBoxLinuxAdditions.run --nox11;
+        expect {
+            "Do you wish to continue" { exp_send "yes\r" ; exp_continue }
+            eof
+        }
+    '
+    sudo umount /mnt
+    sudo rm *.iso
```

Comment:
Let us not be lazy here and only remove the `.iso` that we downloaded.  Set `VBoxGuestAdditions_$VBOX_VERSION.iso` to a variable and use it for wget, mount and in here.